RFC-000-Process-description.md

```diff
@@ -0,0 +1,32 @@
+# Process description
+
+- RFC-Number: 000
+- Status: Draft
+- Discussion-issue: -
+- Created on: 25 Jan. 2019
+
+<!-- TOC GOES here -->
+
+## Description
+
+This document describes the RFC (Request-for-comments) process that is used to document and specify the COMIT procotol.
+The reason for using an RFC process for this is that we would like the documentation itself and the process around it to be open-source and accessible to anyone.
+
+## Lifecycle
+
+### Everything starts as `Draft`
+
+Anybody is welcome to submit an RFC to extend or change the COMIT protocol by opening a pull-request to this repository.
+All accepted changes will have status `Draft`.
```

Comment:
❓ what do you mean? Do you mean all merged Pull Requests?

Reply:
Yes. A merged PR = change to the protocol.